docs/source/index.rst

```diff
@@ -27,6 +27,7 @@ modularity and versatility for a broad set of industry use cases.
    getting_started
    developapps/developing_applications
    tutorials
+	 deployment_guide_overview
```

Comment:
I was thinking that the deployment guide would be a first class topic at the same level as the Operations Guide and not under tutorials. Maybe you had a different thought?

Reply:
That's what I intended. The formatting here just got screwed up because I hit "enter" at the end of the previous line instead of using spaces.